docs/guides/installation_uv.md

```diff
@@ -0,0 +1,260 @@
+
+### Reading a `pyproject.toml` from an Existing Project
+
+If you already have a `pyproject.toml` file—for example, when cloning an existing project—you can use the `uv sync` command to synchronize and install the dependencies defined within it:
+
+```bash
+uv sync
+uv run marimo edit hi.py
+
+```
+
+
+
+This command ensures that your environment matches the dependency specifications of the existing project, making it simple to get up and running without manually adding packages.    
+
+
+## Temporary Installation
+
+When you execute a command using `uv tool run`, no virtual environment folder is created in your working directory. Instead, `uv` takes the following steps:
+
+1. **Caches dependencies or reuses existing cached ones** based on the metadata in the Marimo notebook.
+2. **Creates a temporary virtual environment** on your system.
+3. **Deletes the temporary environment** immediately after the process ends.
+
+This lightweight approach keeps your workspace clean while still providing an isolated, dependency-managed environment for running commands.
+
+For instance, you can run:
+
+```bash
+uv tool run marimo edit hi.py
+```
+
+During the session, you can install additional packages, but their information will not be preserved once the session ends. This is where notebook metadata comes into play, as explained in the following section.
+
+---
+
+### Specifying Additional Requirements
+
+
+#Todo: check how similar this is to https://docs.marimo.io/guides/editor_features/package_management.html#package-management
+
+To include dependency information to the metadata, use the --sandbox option:
+
+```bash
+uv tool run marimo edit hi.py --sandbox
+```
+
+While working in the notebook, you can install packages through pop-ups or via the **Packages** tab. However, **adding packages through the terminal is not supported in sandbox mode.** *(#TODO: Fact-check this limitation.)*
+
+---
+
+### Fully Self-Contained Notebooks
+
+A unique feature of this setup is that the notebook becomes **fully self-contained** and reproducible by anyone. This is achieved by embedding package metadata directly within the notebook, following the guidelines of [PEP 723 – Inline Script Metadata](https://peps.python.org/pep-0723/). 
+
+When opened in a plain text editor, the notebook displays the following embedded metadata: *(#TODO: Replace this example with a screenshot from a text editor. and add a green circle around the metadata)*
+
+```python
+# /// script
+# requires-python = ">=3.11"
+# dependencies = [
+#     "polars==1.16.0",
+# ]
+# ///
+
+import marimo
+
+__generated_with = "0.9.31"
+app = marimo.App(width="medium")
+
+
+@app.cell
+def __():
+    import polars
+    return (polars,)
+```
+
+Once the session ends, the venv is cleared.  
+To reconstruct the session, simply run:
+
+```bash
+uv tool run marimo edit hi.py --sandbox
+```
+
+Marimo detects the embedded dependencies in sandbox mode, so you can also use:
+
+```bash
+uv tool run marimo edit hi.py
+```
```

Comment:
you can delete this part, its not too informational / adds confusion 
````
Marimo detects the embedded dependencies in sandbox mode, so you can also use:

```bash
uv tool run marimo edit hi.py
```
```